math/mathcore/src/VectorizedTMath.cxx

```diff
@@ -1,22 +1,17 @@
-#ifndef ROOT_VectorizedTMath
-#define ROOT_VectorizedTMath
-
-#include "Rtypes.h"
-#include "Math/Types.h"
-#include "TMath.h"
+#include "VectorizedTMath.h"
 
 #if defined(R__HAS_VECCORE) && defined(R__HAS_VC)
 
 namespace TMath {
 ////////////////////////////////////////////////////////////////////////////////
-::ROOT::Double_v Log2(::ROOT::Double_v &x)
+inline ::ROOT::Double_v Log2(::ROOT::Double_v &x)
```

Comment:
Functions inside source files cannot be inline. When the compiler compiles them, they will have a symbol in the object file (and library). When including a header, you have to call the already compiled function, that is in the library.